src/ralph/models/xapi/base.py

```diff
@@ -0,0 +1,19 @@
+"""Base xAPI model"""
+
+from datetime import datetime
+from typing import Literal
+
+from pydantic import BaseModel
+
+from ralph.models.xapi.fields import ActorField, ContextField
+
+from .constants import VERSION
+
+
+class BaseXapiModel(BaseModel):
+    """Base model for xAPI events."""
+
+    actor: ActorField
+    context: ContextField
+    version: Literal["1.0.3"] = VERSION
```

Comment:
I don't clearly understand the point of assigning a `Literal` type with the version value and then the `VERSION` constant.

Reply:
The Idea is that if we don't specify the version - it's value will be "1.0.3" and if we specify the version we check that the value is "1.0.3"

Reply:
Unfortunately python don't accept to do something like:
`version: Literal[VERSION] = VERSION`

Reply:
It means that each time the version changes, you have to change the `Literal` field too?

Reply:
Alright. It will be also the case for other attributes you have defined in fields of `fields.py`

Reply:
Updated this) Just I have kept the xAPI extension related attributes as strings as we don't use them as Literal values.